docs/UserGuide.md

```diff
@@ -1,125 +1,151 @@
 # RMD User Guide:
 
-## Pre-requires
+## Prerequisite
 
-Please read the [requirements][requirements].
+To use RMD, the users need to meet several requirements for both hardware and
+software. Please read the [this doc](../docs/Prerequisite.md) for more details.
 
-[requirements]: https://github.com/intel/rmd/blob/master/docs/PreRequirment.md
+## Installation
```

Comment:
This could be extended as 2 parts:

Install from binary (copy to system path manually)

and 

Install from source code (make, sudo make install)

shall we cover all of them?